pkg/action/custom_resource_lister.go

```diff
@@ -0,0 +1,165 @@
+package action
+
+import (
+	"context"
+	"fmt"
+
+	internal "github.com/operator-framework/kubectl-operator/internal/pkg/action" // TODO move this file here
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/api/pkg/operators/v2alpha1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// OperatorListOperands knows how to find and list custom resources given a package name and namespace.
+type OperatorListOperands struct {
+	config      *internal.Configuration
+	PackageName string
+}
+
+func NewOperatorListOperands(cfg *internal.Configuration) *OperatorListOperands {
+	return &OperatorListOperands{
+		config: cfg,
+	}
+}
+
+func (o *OperatorListOperands) Run(ctx context.Context) (*unstructured.UnstructuredList, error) {
+	opKey := types.NamespacedName{
+		Name:      o.PackageName,
```

Comment:
Don't operator names include the namespace?
```suggestion
		Name:      fmt.Sprintf("%s.%s", o.PackageName, o.config.Namespace),
```